src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts

```diff
@@ -133,6 +133,10 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
   }
 
   onSubmit() {
+    if (this.entryForm.value.activity_id === '-1') {
```

Comment:
The empty option has an id **-1** ?

Reply:
Yes, it was done in an old PR to solve another problem: https://github.com/ioet/time-tracker-ui/blob/10e7694a56bd6cca2df98664d54175bdfa293f86/src/app/modules/time-clock/components/entry-fields/entry-fields.component.html